src/main/java/org/kuali/kfs/pdp/batch/service/impl/Iso20022FormatExtractor.java

```diff
@@ -1895,6 +1908,7 @@ private static void writeMessageToFile(
              * of changing/removing the namespace prefixes, rather than performing String.replaceAll() substitution.
              * Also added setup of a CU-specific EscapeHandler to customize the XML-escaping process.
              */
+            LOG.info("writeMessageToFile writing file " + filename);
```

Comment:
I would suggest modifying this log statement to use a placeholder for inserting the filename.

Reply:
The placeholder suggestion for this log statement has not been implemented yet.